Mathlib/Topology/Basic.lean

```diff
@@ -979,6 +985,10 @@ theorem Filter.HasBasis.mapClusterPt_iff_frequently {ι : Sort*} {p : ι → Pro
 theorem mapClusterPt_iff : MapClusterPt x F u ↔ ∀ s ∈ 𝓝 x, ∃ᶠ a in F, u a ∈ s :=
   (𝓝 x).basis_sets.mapClusterPt_iff_frequently
 
+theorem MapClusterPt.frequently (h : MapClusterPt x F u) {p : X → Prop} (hp : ∀ᶠ y in 𝓝 x, p y) :
+    ∃ᶠ a in F, p (u a) :=
+  mapClusterPt_iff.mp h (setOf p) hp
```

Comment:
Could you add the `ClusterPt` variant? And, ideally, also fix the inconsistency between [clusterPt_iff](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Basic.html#clusterPt_iff) and [mapClusterPt_iff](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Basic.html#mapClusterPt_iff) ?

Reply:
Like this?

Reply:
Exactly! :)